src/components/AppComponents/LanguageToggle/index.tsx

```diff
@@ -23,6 +23,9 @@ const LanguageToggle = props => {
         '@BASECLASS!DATAPAGE!D_LISTREFERENCEDATABYTYPE',
         'HMRC-CHB-WORK-CLAIM!CASE!CLAIM'
       ]);
+
+      PCore.getPubSubUtils().publish('languageToggleTriggered', {language: lang});
+      PCore.getPubSubUtils().publish('languageToggleLocale', []);
```

Comment:
why we are publishing multiple events, we can listen the same event on multiple location.

Reply:
I have added PCore.getPubSubUtils().publish('languageToggleLocale', []); event to check and store localeReference values.